packages/cli/src/commands/console.js

```diff
@@ -0,0 +1,34 @@
+import path from 'path'
+import repl from 'repl'
+
+import { getPaths } from 'src/lib'
+
+/**
+ * Console is less technical and more Railsy
+ */
+export const command = 'console'
+/**
+ * @todo
+ * Appropriate alias?
+ */
+export const aliases = ['c']
+
+/**
+ * @todo
+ * Should we make this more console related?
+ * I.e. does using the word REPL here contradict our efforts to be less technical?
+ */
+export const description = 'Start the Redwood REPL'
+
+const mapDBToContext = (ctx) => {
+  const { db } = require(path.join(getPaths().api.lib, 'db'))
+  ctx.db = db
```

Comment:
It could be a useful feature addition to enable a flag like

`yarn rw console --log`

and the use Prisma middleware to output stats 

https://www.prisma.io/docs/concepts/components/prisma-client/middleware/logging-middleware

Reply:
Oooh, that could be interesting, but maybe something like `--sql` and have it print out SQL statements?

Reply:
`--log` may be a bit too generic.

Reply:
This sounds cool; will definitely list this an idea/to-do

Reply:
Listed in #1556